place_tracker/lib/place_map.dart

```diff
@@ -47,7 +50,7 @@ class PlaceMapState extends State<PlaceMap> {
 
   LatLng _lastMapPosition;
 
-  Map<Marker, Place> _markedPlaces = Map<Marker, Place>();
+  final Map<Marker, Place> _markedPlaces = <Marker, Place>{};
```

Comment:
nit: The type can be [inferred](https://dart.dev/guides/language/sound-dart#type-inference) with just `final _markedPlaces = <Marker, Place>{};` or `final Map<Marker, Place> _markedPlaces = {};`

Reply:
I believe the current style (at least as enforced by the lint checks) is to turn method locals into `var`s, but to leave class fields as fully typed.

Reply:
I would vote for the second option then.

Reply:
Wanna spin up a new PR with your suggestion? This PR has landed and I've already deleted the source repo =)